GameData/RealismOverhaul/Engine_Configs/X405_Config.cfg

```diff
@@ -24,15 +24,15 @@
 //
 //	Dry Mass: 219 Kg
 //	Thrust (SL): ??? kN
-//	Thrust (Vac): 156.3 kN
-//	ISP: ??? SL / 311.9 Vac
+//	Thrust (Vac): 150.5 kN
+//	ISP: ??? SL / 300 Vac
 //	Burn Time: 245
 //	Chamber Pressure: ??? MPa
 //	Propellant: LOX / Kerosene
 //	Prop Ratio: 2.20
 //	Throttle: N/A
 //	Nozzle Ratio: 25
-//	Ignitions: 1
+//	Ignitions: 3
```

Comment:
This reignites the debate as to whether 2 versions of the X-405H may have existed, one with 311 seconds ISP and 3 restarts, and a simplified one with 300 seconds ISP and no restarts. The limited sources I have found show pictures of two distinct X-405Hs with slightly different designs.

Reply:
That's an interesting debate. This is the full excerpt from Sutton:
> [...]the 405H Vega upper-stage engine using LOX/RP-1 propellants. This engine is shown in Fig. 7.3-9. It had a higher  thrust than the Vanguard  booster  engine  (33,840  lbf),  a  larger  nozzle  area  ratio  (25:1),  an  altitude specific  impulse  of  300  s,  and  a  gimbaled  TC.  It  allowed  the  fuel  flow  to  338 History  of  Liquid  Propellant  Rocket  Engines  be  controlled  by  +/-  8%  in  order  to  fully  utilize  all  of  the  propellant.  **The  supply  subsystem for hypergolic ignition fluid was altered to  allow  three altitude/vacuum  starts.**  

The only thing I can find in a very quick google search, from the Smithsonian Air And Space Museum website, seems to back this up...

Reply:
What should the second variant be then? Because there is no second version in the configs as of when I made the change to 3 ignitions. the 405H goes, right now:

405H : 150V_Thrust, 300V_Isp, 3 ignitions
405H-3 : 161V_Thrust, 323V_Isp, 1 ignition
405-H-4 : 186V_Thrust, 323V_Isp, 1 ignition

I could create just two versions of the first, one with 311isp 1 ignition and one with 300isp 3 ignitions? Not sure what the name would be then, however...

Reply:
Well, it seems like some of the confusion (on my end) was that the comments were different from the actual config when I started, and I did not notice. the -3 and -4 variants both actually have 3 ignitions already in config. However, Sutton only mentions a 1958 contract for the "405H Vega upper-stage engine," with no mention of variants, which leads me to believe he was talking about the base variant.

Reply:
Oh, I see. Yeah, I dunno here. If we think all variants should have the 3 ignitions, then it's just a straightforward Isp increase at each level of config.

Looking at the actual config, the H-3 and H-4 are listed as "speculative" so I would leave them alone and just apply any changes to the regular X-405H. That said, I would tend to think we should keep the data from the Vega Final Report (direct from Convair) rather than Sutton's numbers, for Vega?

Reply:
I intentionally skipped designations from the real X-405H to the speculative H-3, leaving space for an H-2 should it turn out that there were two variants of the X-405H.

From my understanding, the original X405H was intended to have 3 ignitions, and 311 ISP, and an engine of these specifications was at least partially constructed. However, when Juno IV and Atlas-Vega were rolled into one project, Vega no longer required as much performance and restarts, because the Juno upper stage would do most of the work instead, resulting in the 1 ignition 300 ISP X405H that was actually mounted on Vega. Should Atlas-Vega have survived, it is conceivable the original X405H design be revived for more payload, creating the X405H-2, with 311 ISP, 156 kN thrust, and 3 ignitions.